packages/brain/src/index.ts

```diff
@@ -103,17 +102,33 @@ const proofOfValidationCron = new CronJob(shareCronInterval, () =>
 );
 proofOfValidationCron.start();
 
-// executes once every epoch
-export const trackValidatorsPerformanceCronTask = new CronJob(slotsPerEpoch * secondsPerSlot * 1000, () =>
-  trackValidatorsPerformanceCron({ brainDb, postgresClient, beaconchainApi, executionClient, consensusClient })
-);
-startWithinTenFirstPercentageOfEpoch({
-  minGenesisTime,
-  secondsPerSlot,
-  slotsPerEpoch,
-  jobFunction: trackValidatorsPerformanceCronTask
+// execute the performance cron task every 1/4 of an epoch
+const secondsPerEpoch = slotsPerEpoch * secondsPerSlot;
+const oneFourthEpochInSeconds = secondsPerEpoch / 4;
+let lastProcessedEpoch: number | undefined = undefined;
```

Comment:
Why not moving 
```ts
let lastProcessedEpoch: number | undefined = undefined;
```

to the file `trackValidatorsPerformanceCron` and do there the logic of 
```ts
    const currentEpoch = await beaconchainApi.getEpochHeader({ blockId: "finalized" });

    if (currentEpoch !== lastProcessedEpoch) {
      await trackValidatorsPerformanceCron({
        brainDb,
        postgresClient,
        beaconchainApi,
        executionClient,
        consensusClient,
        currentEpoch
      });
      lastProcessedEpoch = currentEpoch;
    }
  } catch (error) {
    logger.error(`Failed to fetch or process epoch:`, error);
  }
  ```
  
  This way the `index.ts` would look cleaner. You can wrap this logic in a new function and call just this function in the cron

Reply:
done